examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    # dv.grad(dv.div(v)) does not get flattened and will need to be explicitly written out
</pre><p>Not sure what that means but does not make sense to have to go to those sympy matrices. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Maybe I'm overthinking? I was under the impression that `dv.grad(dv.div(v))` is going to result in bigger stencils than doing the grad of the divergence algebraically and implementing that